### DISCUSSION TITLE
iets van functionaliteit om overlappende polygonen te kunnen onderzoeken/zien wat eronder ligt

### DISCUSSION THREAD

**Discussion by jipclaassens** [ObjectVision/GeoDMS#48] [Ideas]
nu klikken op polygonen met overlap werkt niet altijd lekker

Maarten Hilferink [28 Feb 2022 at 16:59](https://trello.com/c/Qv6XQ3QW/54-iets-van-functionaliteit-om-overlappende-polygonen-te-kunnen-onderzoeken-zien-wat-eronder-ligt#comment-621cf1559039f019d52da368)
inversion of drawing order

[Reply](https://trello.com/c/Qv6XQ3QW/54-iets-van-functionaliteit-om-overlappende-polygonen-te-kunnen-onderzoeken-zien-wat-eronder-ligt#)
Maarten Hilferink [28 Feb 2022 at 16:58](https://trello.com/c/Qv6XQ3QW/54-iets-van-functionaliteit-om-overlappende-polygonen-te-kunnen-onderzoeken-zien-wat-eronder-ligt#comment-621cf11b79db997539a27104)
temp transparency van overlappend polygoon vanuit layer control

[Reply](https://trello.com/c/Qv6XQ3QW/54-iets-van-functionaliteit-om-overlappende-polygonen-te-kunnen-onderzoeken-zien-wat-eronder-ligt#)
Maarten Hilferink [28 Feb 2022 at 16:56](https://trello.com/c/Qv6XQ3QW/54-iets-van-functionaliteit-om-overlappende-polygonen-te-kunnen-onderzoeken-zien-wat-eronder-ligt#comment-621cf0b333c84f76a6ddea7c)
meerdere values van over elkaar liggende polygonen in dezelfde layer tonen als mini-tabel in de layer control.

[Reply](https://trello.com/c/Qv6XQ3QW/54-iets-van-functionaliteit-om-overlappende-polygonen-te-kunnen-onderzoeken-zien-wat-eronder-ligt#)
Maarten Hilferink [28 Feb 2022 at 16:55](https://trello.com/c/Qv6XQ3QW/54-iets-van-functionaliteit-om-overlappende-polygonen-te-kunnen-onderzoeken-zien-wat-eronder-ligt#comment-621cf0832da9176d4acc48f9) (edited)
alle values van verschillende layers in controls tonen (muv topografische lagen, inactief of WMS)

**Comment by mtbeek32** [comment]
Menu option to select overlapping polygons in one layer.
